python-avd/pyavd/_eos_designs/schema/schema_fragments/wan_use_agent_env_var_for_kernel_software_forwarding_ecmp.schema.yml

```
@@ -0,0 +1,19 @@
+# Copyright (c) 2023-2025 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../_schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  wan_use_agent_env_var_for_kernel_software_forwarding_ecmp:
+    type: bool
+    documentation_options:
+      table: wan-settings
+    # TODO: AVD 6.0.0 Consider changing default to false
+    default: true
+    description: |-
+      This specifies the preferred mechanism to program ECMP routes within the kernel for software forwarding.
+      When enabled, configures an environment variable to restart KernelFib agent for ECMP route programming.
+      When disabled, ECMP route programming in the kernel is achieved without agent restart.
+      This is relevant to Pathfinder and AUTOVPN designs. This is supported on EOS version 4.33.2 and later.
```

@ashenoy-arista this seems to indicate that the ENV VAR is supported starting 4.33.2 which is not the case but actually the contrary.

Basically before 4.33.2 this knob should always be set to true right?

```suggestion
      This is relevant to Pathfinder and AUTOVPN designs. This is supported on EOS version 4.33.2 and later.
```

---

Updated the description